src/shared/modules/commands/commandsDuck.js

```diff
@@ -161,7 +161,13 @@ export const handleCommandEpic = (action$, store) =>
       store.dispatch(clearErrorMessage())
       const maxHistory = getMaxHistory(store.getState())
       store.dispatch(addHistory(action.cmd, maxHistory))
-      const statements = shouldEnableMultiStatementMode(store.getState())
+
+      // Semicolons in :style grass break parsing of multiline statements from codemirror.
+      const useMultiStatement =
```

Comment:
The best way I've found to solve the issue of `extractStatementsFromString` from cypher-codemirror can't parse grass is to avoid using it for style commands. We can't run style and some other command so it makes sense not to use multi-statement anyway. I'm not 100% happy with the magic string of ":style" and it's special handling to other commands though :/

Reply:
Im not sure how 'special' situations are handled in Browser or if at all. The only comment I would have is whether a `@todo` comment here makes sense too or adding `:style` to some variable that holds non multi statement commands? May not be relevant, Im not sure

Reply:
When the allow multi statement setting is turned on, some commands (including :style) can't be part of a multistatement but should still work on their own. The problem is that :style is parsed in such a way that browser thinks it's multiple statments, since cypher-code mirror util we have can't parse grass to split the commands properly. So i don't think the variable would be helping and I'm not sure what to even write in a todo comment.. I'll write it down and discuss with hane/emil when they get back